pkg/apis/serving/v1alpha1/configuration_lifecycle.go

```diff
@@ -24,6 +24,11 @@ import (
 
 var confCondSet = apis.NewLivingConditionSet()
 
+// GetConditionSet retrieves the ConditionSet of the Configuration. Implements the KRShaped interface.
+func (c *Configuration) GetConditionSet() apis.ConditionSet {
```

Comment:
```suggestion
func (*Configuration) GetConditionSet() apis.ConditionSet {
```

no need to name the Configuration for these here and for others